test/unit/specs/Deck.spec.js

```diff
@@ -168,13 +239,84 @@ describe('Deck.js', () => {
     expect(hackCounter).to.equal(9)
   })
 
+  // test the number of Virus cards in the deck 3 player
+  it('test the number of virus cards in the deck', () => {
+    let testDeck = new Deck()
+    testDeck.initDeck(3)
+    let cardsArray = testDeck.cards
+    let counter = 0
+    for (let idx of cardsArray) {
+      if (idx.type === 'VIRUS') {
+        counter++
+      }
+    }
+    expect(counter).to.equal(9)
+  })
+
+  // test the number of Power Outage cards in the deck 3 player
+  it('test the number of power outage cards in the deck', () => {
+    let testDeck = new Deck()
+    testDeck.initDeck(3)
+    let cardsArray = testDeck.cards
+    let counter = 0
+    for (let idx of cardsArray) {
+      if (idx.type === 'POWEROUTAGE') {
+        counter++
+      }
+    }
+    expect(counter).to.equal(9)
+  })
+
+  // PROTECTION CARDS
```

Comment:
ANTIVIRUS?

Reply:
This comment was just to make it a bit easier to read when I was writing the tests. Ill take it out.

Reply:
No, it looked like a copy/paste error. The tests have to do with ANTIVIRUS not PROTECTION. The markers are a good idea.

Reply:
Oh I see, I used Protection to refer to the 3 safety cards that protected you from all attack cards.

Reply:
Oh, OK.